server/index.js

```diff
@@ -28,38 +21,6 @@ function register (server, options, next) {
   // mapreduce is required for `db.query()`
   PouchDB.plugin(require('pouchdb-mapreduce'))
```

Comment:
I followed your instructions and I just did 'npm test'. But it still has an error related to variable 'config.PouchDB.preferredAdapters'. I have no idea how to deal with this variable which could make an error.
The error messages are the same like this below. 
`not ok TypeError: Cannot read property 'preferredAdapters' of undefined`